src/main/java/org/mitre/synthea/engine/ReactionProbabilities.java

```diff
@@ -0,0 +1,120 @@
+package org.mitre.synthea.engine;
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.apache.commons.math3.distribution.EnumeratedDistribution;
+import org.apache.commons.math3.util.Pair;
+import org.mitre.synthea.world.agents.Person;
+import org.mitre.synthea.world.concepts.HealthRecord;
+
+/**
+ * ReactionProbabilities is a structure to capture the kind and severity of reaction for a
+ * given AllergyIntolerance.
+ */
+public class ReactionProbabilities implements Serializable {
```

Comment:
Just a note that I was initially a little confused by jumping back & forth between reaction "severities" used in some places and reaction "probabilities" used here. Would be nice to have those a little more consistent but not a big deal.